### PR TITLE
Ignore the files from uploading

### PR DESCRIPTION
**Pull request by kumarprakash1636**
If applied, this commit will ignore the following files:
file11.txt
file13.txt
file14.txt
file21.txt
file22.txt
file23.txt
file24.txt
file31.txt
file32.txt
file33.txt
file34.txt
(Above files are in .gitignore file)

file2.txt(Not in .gitignore file)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,7 @@
+file1?.txt
+file2?.txt
```
Comment (sudarshan-dodiya) [inline .gitignore]: Why is the rule for file2 here? Doesn't it state that the rule should not be there?

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+file3?.txt
+!file35.txt
+
```
Comment (sudarshan-dodiya) [inline .gitignore:4]: Here, you have not added git ignore rules for any of the folders?

**Review by sudarshan-dodiya — CHANGES_REQUESTED** [review]
The title and commit messages do not conform to the format for writing a good commit message: https://coda.io/d/_dHOXTAtFBHO/Creating-Pull-Requests_sulg3#_luNjv

File: .gitignore (hunk 1 above)
Comment (kumarprakash1636) [reply .gitignore]: This rule is not targetting file2.txt

File: .gitignore (hunk 2 above)
Comment (kumarprakash1636) [reply .gitignore:4]: Done

Comment (sudarshan-dodiya) [reply .gitignore:4]: How are you specifying here to not ignore file12.txt?

**Review by sudarshan-dodiya — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 2 above)
Comment (kumarprakash1636) [reply .gitignore:4]: Solved

Comment (sudarshan-dodiya) [reply .gitignore:4]: It was not really required. You should not assume that if a reviewer is commenting something, a change needs to be made. If your implementation works, then you should clarify that what you have done is correct.

**Review by sudarshan-dodiya — APPROVED** [review]

**Pull request merged**